agent-packaging/packaging-scripts/run_daisy_workflow.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+./daisy -project gcp-guest -zone us-central1-c -var:gcs_path=gs://osconfig-agent-package ./build_packages.wf.json
```

Comment:
this is another file that belongs in `guest-test-infra`, and renamed to main.sh

maybe we can take the JSON as an input, so this docker image could be used for building packages out of multiple repos?

Reply:
same as above